apps/example-orchestrator-ui/components/Subscription/Subscription.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { FC, useMemo, useState } from 'react';
+import {
+    ProcessesTimeline,
+    SubscriptionActions,
+    SubscriptionDetailTree,
+    SubscriptionGeneral,
+} from '@orchestrator-ui/orchestrator-ui-components';
+import {
+    EuiBadge,
+    EuiFlexGroup,
+    EuiFlexItem,
+    EuiTab,
+    EuiTabs,
+    EuiText,
+} from '@elastic/eui';
+import { useQuery } from 'react-query';
+import { GRAPHQL_ENDPOINT } from '../../constants';
+import { GraphQLClient } from 'graphql-request';
+import {
+    GET_SUBSCRIPTION_DETAIL_COMPLETE,
+    GET_SUBSCRIPTION_DETAIL_OUTLINE,
+    mapApiResponseToSubscriptionDetail,
+} from './subscriptionQuery';
+import { SubscriptionContext } from '@orchestrator-ui/orchestrator-ui-components';
+import { getColor, tabs } from './utils';
+
+type SubscriptionProps = {
+    subscriptionId: string;
+};
+
+const graphQLClient = new GraphQLClient(GRAPHQL_ENDPOINT);
+
+export const Subscription: FC<SubscriptionProps> = ({ subscriptionId }) => {
+    const { setSubscriptionData, loadingStatus } =
+        React.useContext(SubscriptionContext);
+
+    // Tab state
+    const [selectedTabId, setSelectedTabId] = useState(
+        'service-configuration-id',
+    );
+    const selectedTabContent = useMemo(() => {
+        // @ts-ignore: todo -> improve tabs, refactor them to separate component
+        return tabs.find((obj) => obj.id === selectedTabId)?.content;
+    }, [selectedTabId]);
+    const onSelectedTabChanged = (id: string) => {
+        setSelectedTabId(id);
+    };
+
+    // GUI state done, deal with data:
+
+    // Todo: Find out if pre fetch can be used again. The shape of table cache seems to have changed
```

Comment:
The status of the branch is Draft. I use todo's to keep track of progress and to not forget stuff.

I will probably create a ticket for this issue: something in the Grid branch / Data loading MR : did break this feature.
(which uses the already cached data, which will be often available because the user visited the subscription before going to detail page)

Reply:
Converted into #97